src/main/java/spireCafe/interactables/patrons/councilofghosts/CouncilOfGhostsCutscene.java

```diff
@@ -0,0 +1,128 @@
+package spireCafe.interactables.patrons.councilofghosts;
+
+import basemod.cardmods.InnateMod;
+import basemod.helpers.CardModifierManager;
+import com.badlogic.gdx.math.MathUtils;
+import com.evacipated.cardcrawl.mod.stslib.fields.cards.AbstractCard.AutoplayField;
+import com.megacrit.cardcrawl.actions.common.ObtainPotionAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.FontHelper;
+import com.megacrit.cardcrawl.potions.GhostInAJar;
+import com.megacrit.cardcrawl.relics.BurningBlood;
+import com.megacrit.cardcrawl.relics.Sozu;
+import com.megacrit.cardcrawl.vfx.cardManip.PurgeCardEffect;
+import spireCafe.abstracts.AbstractCutscene;
+import spireCafe.abstracts.AbstractNPC;
+import spireCafe.cardmods.AutoplayMod;
+import spireCafe.util.cutsceneStrings.CutsceneStrings;
+import spireCafe.util.cutsceneStrings.LocalizedCutsceneStrings;
+
+import static spireCafe.Anniv7Mod.makeID;
+
+public class CouncilOfGhostsCutscene extends AbstractCutscene {
+    public static final String ID = makeID(CouncilOfGhostsCutscene.class.getSimpleName());
+    private static final CutsceneStrings cutsceneStrings = LocalizedCutsceneStrings.getCutsceneStrings(ID);
+
+    private static final float MAX_HP_COST = 0.4F;
+    private static final float HIGH_ASC_MAX_HP_COST = 0.5F;
+    private static final float HP_COST = 0.1F;
+    private static final float HIGH_ASC_HP_COST = 0.15F;
```

Comment:
No problem. I noticed the Koishi used different values at different Ascensions so I copied that - might be worth getting that one revisited if we're no longer taking that approach. 

I've gone with 50% max HP and 25% HP as suggested. Agreed with the concern about balance - let's keep an eye on this patron as you suggest, but I'm happy with these as initial values.

Reply:
That's funny about Koishi, shows how hard it can be to keep even a small group of people on the same page.

Thanks for pointing this out, I'll follow up with the other maintainers about that and change Koishi as necessary.